Resources/Prototypes/Entities/Objects/Misc/land_mine.yml

```diff
@@ -74,3 +88,17 @@
     intensitySlope: 3
     totalIntensity: 120 # about a ~4 tile radius
     canCreateVacuum: false
+
+- type: entity
+  suffix: armed
+  parent: LandMineExplosive
+  id: LandMineExplosiveArmed
```

Comment:
`LandMineExplosive` should be the armed version because it is used in some maps already.
Same with `LandMineKick`, and `LandMineModular`.
Make a new unarmed prototype for each.

Reply:
yeah it probably would be better than i said

Reply:
Fixed